src/services/__tests__/httpService.test.js

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+
+import * as http from '../httpService';
+
+const defaultHeaders = { abc: 'def' };
+fetch = jest.fn().mockRejectedValue('error');
```

Comment:
same here with the constants and can be separate them a bit, 

mocks above

and constants below?

Reply:
Yup. We need to get started on the testing standards!